src/EPR.Calculator.API/appsettings.Development.json

```diff
@@ -18,9 +18,8 @@
     "PostMessageRetryCount": 3,
     "PostMessageRetryPeriod": 2
   },
-  "AzureBlobStorage": {
-    "ConnectionString": "ConnectionString",
-    "ContainerName": "ContainerName",
-    "CsvFileName": "CsvFileName"
+  "BlobStorage": {
+    "ConnectionString": "DefaultEndpointsProtocol=https;AccountName=devrwdwebsa1403;AccountKey=hbh3PXMlsTbo37VtDFKse2L/S/I2/Rhdis1kL4zymcSLO7zIK/GCIuzicIcvKkZPJZ/jfC8K6QA/+AStX4YG5w==;EndpointSuffix=core.windows.net",
```

Comment:
Please update the property values to something generic.

Reply:
updated